### PR TITLE
chore(readme): add Angular Turkey #NGTurkey

### PR DESCRIPTION
**Pull request by mehmetakifalp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
...
 ---
 
+![Angular Turkey](https://pbs.twimg.com/profile_images/1094861063949074432/sWMXLmv__400x400.jpg)
+
+## Angular Turkey
+
+Angular meetup in Turkey. Monthly meetups and building a collaborative Angular community in Turkey. We're publishing Turkish articles, examples, helpful resource for Turkish developer.
+Right now we have 3 good angular libraries in our GitHub repo.
+
+- Meetup: [ngturkey](https://www.meetup.com/ngturkey/)
```
Comment (Splaktar) [inline README.md:855]:
```suggestion
* URL: [ngturkey](https://www.meetup.com/ngturkey/)
```

File: README.md [hunk 2]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
...
+- Twitter: [@ngTurkiye](https://twitter.com/ngTurkiye)
```
Comment (Splaktar) [inline README.md:856]:
```suggestion
* Twitter: [@ngTurkiye](https://twitter.com/ngTurkiye)
```

File: README.md [hunk 3]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
...
+- GitHub: [@ng-turkey](https://github.com/ng-turkey)
```
Comment (Splaktar) [inline README.md:857]:
```suggestion
* GitHub: [@ng-turkey](https://github.com/ng-turkey)
```

File: README.md [hunk 4]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
...
+- Slack: [ngturkey](https://ngturkey.slack.com)
```
Comment (Splaktar) [inline README.md:858]:
```suggestion
* Slack: [ngturkey](https://ngturkey.slack.com)
```

File: README.md [hunk 5]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
 * URL: [https://www.meetup.com/Angular-Ruhr/](https://www.meetup.com/Angular-Ruhr/)
 ---
 
+![Angular Turkey](https://pbs.twimg.com/profile_images/1094861063949074432/sWMXLmv__400x400.jpg)
+
```
Comment (Splaktar) [inline README.md:849]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
...
+## Angular Turkey
+
+Angular meetup in Turkey. Monthly meetups and building a collaborative Angular community in Turkey. We're publishing Turkish articles, examples, helpful resource for Turkish developer.
+Right now we have 3 good angular libraries in our GitHub repo.
+
+- Meetup: [ngturkey](https://www.meetup.com/ngturkey/)
+- Twitter: [@ngTurkiye](https://twitter.com/ngTurkiye)
+- GitHub: [@ng-turkey](https://github.com/ng-turkey)
+- Slack: [ngturkey](https://ngturkey.slack.com)
+
```
Comment (Splaktar) [inline README.md:859]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
 * URL: [https://www.meetup.com/Angular-Ruhr/](https://www.meetup.com/Angular-Ruhr/)
 ---
 
+![Angular Turkey](https://pbs.twimg.com/profile_images/1094861063949074432/sWMXLmv__400x400.jpg)
+
+## Angular Turkey
+
```
Comment (Splaktar) [inline README.md:851]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -845,3 +845,16 @@ Official account for the #AngularRuhr Meetup by @robinboehm @thiele_leonard and
...
+Angular meetup in Turkey. Monthly meetups and building a collaborative Angular community in Turkey. We're publishing Turkish articles, examples, helpful resource for Turkish developer.
+Right now we have 3 good angular libraries in our GitHub repo.
+
```
Comment (Splaktar) [inline README.md:854]:
```suggestion
* Location: ???, Turkey
```

**Review by Splaktar — CHANGES_REQUESTED** [review]
Some suggestions for consistency

**Comment by webmaxru** [comment]
Was too quick to Merge without applying useful changes by @Splaktar. Fixing

**Pull request merged**